src/wp-admin/includes/class-theme-installer-skin.php

```diff
@@ -283,7 +283,7 @@ private function do_overwrite() {
 		 * @since 5.5.0
 		 *
 		 * @param string   $table              The output table with Name, Version, Author, RequiresWP, and RequiresPHP info.
-		 * @param WP_Theme $current_theme_data Current theme data.
+		 * @param WP_Theme $current_theme_data Currently active theme data.
```

Comment:
This change may be due to the name of the variable. However, there are other variables in this PR that contain `$current_` but have had their description changed from "Current" to "Active".

Whether they are described as "Active..." or "Currently active..." is a matter of choice, but the choice should be consistent.

Reply:
It was not because of the name of the variable, but rather to help distinguish from the $new_theme_data variable. But I think it would be fine without "Currently" :)